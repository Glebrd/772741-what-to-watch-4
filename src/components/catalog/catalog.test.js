import React from "react";
import renderer from "react-test-renderer";
import Catalog from "../catalog/catalog";

const movies = [];
const onCardTitleClick = () => {};

it(`Catalog renders correctly`, () => {
  const tree = renderer
    .create(
        <Catalog
          movies={movies}
          onCardTitleClick={onCardTitleClick}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
