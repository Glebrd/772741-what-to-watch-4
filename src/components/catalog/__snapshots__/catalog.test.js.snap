// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Catalog renders correctly 1`] = `
<div
  className="catalog__movies-list"
/>
`;
